dmutils/filters.py

```diff
@@ -7,7 +7,7 @@
 
 
 def markdown_filter(text, *args, **kwargs):
-    return markdown(text, ['markdown.extensions.abbr'], *args, **kwargs)
+    return markdown(text, *args, **dict(kwargs, extensions=['markdown.extensions.abbr']))
```

Comment:
This was required because the dmutils tests were failing due to an API change with the python markdown package.